tests/unit/fixtures/sample-collection.js

```diff
@@ -2453,6 +2496,9 @@ module.exports = {
                                 name: "Cookies",
                                 originalRequest: {
                                     header: [],
+                                    url: {
```

Comment:
@kunagpal might be a good idea to make sure this does not happen, minimizes surface area for potential bugs